base/mem_trie.hpp

```diff
@@ -48,20 +55,99 @@ class MapMoves
     return *node;
   }
 
+  void AddSubtree(Char const & c, std::unique_ptr<Subtree> subtree)
+  {
+    ASSERT(!GetSubtree(c), ());
+    m_subtrees.emplace(c, move(subtree));
+  }
+
   void EraseSubtree(Char const & c) { m_subtrees.erase(c); }
 
-  bool Empty() const { return m_subtrees.empty(); }
+  size_t Size() const { return m_subtrees.size(); }
```

Comment:
`#include <cstddef>`

Reply:
??? cstddef is already included. Please, take a look at the top of the file.

Reply:
Oh! Yes. Sorry.